src/styles/index.css

```diff
@@ -0,0 +1,85 @@
+.page {
+  margin: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+}
```

Comment:
It is better to place general styles like `.page` in a separate file. For example, you can create a `general.scss` file and import it into your `main.scss` file. This will make your code more structured and easier to maintain.